src/viewer.js

```diff
@@ -6,19 +6,59 @@ var Viewer = function (el) {
   this.el.classList.add('kokorigami-viewer');
   this._ = {
     model: new Model(),
-    progress: 0,
+    frame: 0,
     renderer: new Renderer(this.el)
   };
   return this;
 };
 
 Viewer.prototype = {};
 
+Viewer.prototype.render = function () {
+  this._.renderer.play();
+};
+
+Viewer.prototype.play = function (frame) {
+  if (this.raf) cancelAnimationFrame(this.raf);
```

Comment:
Viewer now tracks the animation frame and cancels it when play is called in the middle of an existing animation, so this should solve the issues around invoking play functions multiple times.